src/components/time/time.tsx

```diff
@@ -3,14 +3,19 @@ import { Link } from 'react-router-dom';
 import { convertTimestampToString } from 'src/utils/date';
 import styles from './time.module.scss';
 
-function Time({ msTime, linkTo }: { msTime: number; linkTo: string }) {
+function Time({ msTime, linkTo }: { msTime: number; linkTo?: string }) {
   const [valueTime, prefixTime] = convertTimestampToString(msTime).split(' ');
+  let Component: HTMLSpanElement | Link = 'span';
+
+  if (linkTo) {
+    Component = Link;
+  }
 
   return (
-    <Link to={linkTo} className={styles.wrapper}>
+    <Component to={linkTo} className={styles.wrapper}>
```

Comment:
link not connected to time component, if need to reuse this logic, better create 2 components